scripts/ci/check/version_test.go

```diff
@@ -21,22 +21,35 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"regexp"
+	"runtime"
 	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 	"golang.org/x/mod/modfile"
 )
 
-const GoVersion = "1.20"
+const (
+	GoVersion = "1.20"
+	CPUType   = "64"
+)
 
 func TestGoVersion(t *testing.T) {
-	goversion, err := exec.Command("go", "version").Output()
-	require.NoError(t, err)
+	types := runtime.GOARCH
+
+	ok := strings.Contains(types, CPUType)
```

Comment:
According to https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63#goarch-values,

It is not accurate to simply check "64" string. Probably, we can use,

```go
const PtrSize = 4 << (^[uintptr](https://pkg.go.dev/builtin#uintptr)(0) >> 63)
```

From godoc,

> PtrSize is the size of a pointer in bytes - unsafe.Sizeof(uintptr(0)) but as an ideal constant. It is also the size of the machine's native word size (that is, 4 on 32-bit systems, 8 on 64-bit).

Reply:
Ok. I have tried it on a 64-bit system and it is coming out as 8. I will try it on a 32-bit system or an i386 version of Go to check if it's coming as 4 and then push the new commit.

Reply:
@lujiajing1126 I have added this snippet and also tested it with both a 64-bit and a 32-bits system. The output is coming out as desired when running the test.